### PR TITLE
Reset to react-from-html

### PR DESCRIPTION
**Pull request by chrisvxd**
Strips things back to a clean-isn project. See commit history for changes.

Unsure about removal of Travis.

Other considerations:

1. Break out of monorepo. The monorepo definitely adds some overhead, and npm package README confusion / duplication.
2. Add GitHub actions to replace Travis, although probably issueify for a later date.
3. Deliberately avoiding any NPM namespace since it's not measured-specific, and I think `react-from-html` is available.
4. Need to get ownership of `dom-element-to-react` package, rename it or kill it if we fold out of monorepo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,13 @@
 {
-  "name": "react-from-markup-workspace",
+  "name": "react-from-html-workspace",
   "version": "0.0.0",
   "main": "index.js",
-  "author": "Simon Andrews <me@simonandrews.ca>",
+  "author": "Measured.co <hello@measured.co>",
```
Comment (monospaced) [inline package.json]: `Measured Co.` might be better, as standard way to write org name.

File: packages/react-from-markup/package.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
-  "name": "react-from-markup",
-  "version": "2.0.0-pre.5",
+  "name": "react-from-html",
+  "version": "0.1.0",
   "description": "Convert regions of markup into React, with custom callbacks",
   "main": "lib/index.js",
-  "author": "Simon Andrews <me@simonandrews.ca>",
+  "author": "Measured.co <hello@measured.co>",
```
Comment (monospaced) [inline packages/react-from-markup/package.json]: As above.

File: .prettierrc.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true
```
Comment (monospaced) [inline .prettierrc.json:3]: Not the biggest fan of `singleQuote` as a customisation, but hey. Do like trailing commas.

Comment (monospaced) [inline .prettierrc.json]: Not the biggest fan of `singleQuote` as a customisation, but hey. Do like trailing commas.

File: packages/react-from-markup/package.json (hunk 2 above)
Comment (chrisvxd) [reply packages/react-from-markup/package.json]: Fixed

File: .prettierrc.json (hunk 3 above)
Comment (chrisvxd) [reply .prettierrc.json]: Switched back

File: package.json (hunk 1 above)
Comment (chrisvxd) [reply package.json]: Fixed, keep tripping up on this.

**Comment by monospaced** [comment]
1. vote break out of monorepo
2. yep, issueify
3. agree, no `@measured` required
4. either move it to a dep (maybe fine for react-from-markup and react-from-html to share it?) OR fold it in.

**Comment by chrisvxd** [comment]
@monospaced have ejected from monorepo. Pending final review.

**Pull request merged**